fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -100,6 +126,17 @@ class DemographicParity(ConditionalSelectionRate):
 
     .. math::
       P[h(X) = y' | A = a] = P[h(X) = y'] \; \forall a, y'
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    a single event, `all`. Consequently, the `prob_event`
+    :class:`pandas:pandas.DataFrame`
+    will only have a single entry, which will be equal to 1.
+    Similarly, the `index` property will have twice as many entries
+    (corresponding to the positive and negative Lagrange multipliers)
+    as there are unique values for the sensitive feature.
```

Comment:
the Lagrange multipliers are always non-negative; how about the following rewrite:
"corresponding to the Lagrange multipliers for positive and negative constraints"

Reply:
Agree that that's a clearer way of putting it.